commit c8ce77d48c18b51de5573fdec3e5d0ad13d6967a (refs/changes/09/709/1)
Author: David Pacheco <dap@joyent.com>
Date:   2016-10-17T09:53:36-07:00 (3 years ago)
    
    MANTA-3005 muskie should adopt moray v2
    MANTA-3006 marlin should adopt moray v2

diff --git a/lib/moray.js b/lib/moray.js
index a73b682..12dbed3 100644
--- a/lib/moray.js
+++ b/lib/moray.js
@@ -479,8 +479,8 @@ Moray.prototype.initAttempt = function initAttempt() {
                 log: self.log,
                 host: self.host,
                 port: self.port,
-                reconnect: true,
-                retry: self.retry
+                retry: self.retry,
+                unwrapErrors: true
             });
 
             var onErrorDuringConnect = function onErrDuringConnect(err) {
diff --git a/package.json b/package.json
index 78ded34..90e09e8 100644
--- a/package.json
+++ b/package.json
@@ -14,7 +14,7 @@
         "backoff": "2.3.0",
         "bunyan": "0.22.1",
         "lru-cache": "2.3.1",
-        "moray": "git+ssh://git@github.com:joyent/node-moray.git#fd5781bc25a9bfe2ba82167664639753fb9f0ca5",
+        "moray": "^2.0.0",
         "once": "1.3.0",
         "restify": "2.6.3",
         "redis": "0.10.1",
